lib/config/presets/internal/default.ts

```diff
@@ -327,6 +327,10 @@ export const presets: Record<string, Preset> = {
     description: 'Raise a PR first before any automerging',
     automergeType: 'pr',
   },
+  pruneBranchAfterAutomerge: {
+    description: 'Prune branch after automerging',
+    enabled: true,
+  },
```

Comment:
remove, as it's enabled by default

Reply:
done

Reply:
please read https://github.com/renovatebot/renovate/blob/main/.github/contributing.md#resolve-review-comments-instead-of-commenting and reduce noise.